runtime/src/lib.rs

```diff
@@ -314,15 +306,17 @@ impl pallet_contracts::Config for Runtime {
 	type CallFilter = frame_support::traits::Nothing;
 	type DepositPerItem = DepositPerItem;
 	type DepositPerByte = DepositPerByte;
+	type CallStack = [pallet_contracts::Frame<Self>; 31];
 	type WeightPrice = pallet_transaction_payment::Pallet<Self>;
 	type WeightInfo = pallet_contracts::weights::SubstrateWeight<Self>;
 	type ChainExtension = ();
 	type DeletionQueueDepth = DeletionQueueDepth;
 	type DeletionWeightLimit = DeletionWeightLimit;
 	type Schedule = Schedule;
-	type CallStack = [pallet_contracts::Frame<Self>; 31];
 	type AddressGenerator = pallet_contracts::DefaultAddressGenerator;
 	type ContractAccessWeight = DefaultContractAccessWeight<RuntimeBlockWeights>;
+	type MaxCodeLen = ConstU32<{ 128 * 1024 }>;
+	type RelaxedMaxCodeLen = ConstU32<{ 256 * 1024 }>;
```

Comment:
Otherwise we would decrease the allowed size.

```suggestion
	type MaxCodeLen = ConstU32<{ 256 * 1024 }>;
	type RelaxedMaxCodeLen = ConstU32<{ 512 * 1024 }>;
```

Reply:
Ok, I had copied those from here: https://github.com/paritytech/substrate/blob/master/bin/node/runtime/src/lib.rs#L1121-L1122. Maybe adapt those as well?

Reply:
No. We have higher values than we would recommend for production in this node because of debug builds.

Reply:
We should add this to the readme then, there's a section of modifications we made that render this node unusable for production. Is there a reasoning behind only recommending the lower values for production?